webhook_server/tests/test_runner_handler.py

```
@@ -1889,6 +1910,398 @@ async def run_command_side_effect(command: str, **kwargs: Any) -> tuple[bool, st
+        mock_pr = Mock()
+        commit1 = Mock()
+        commit1.commit.message = "first commit without signoff"
+        commit1.commit.author.name = "Author One"
+        commit1.commit.author.email = "one@example.com"
+        commit2 = Mock()
+        commit2.commit.message = "second commit\n\nSigned-off-by: Author Two <two@example.com>\n"
+        commit2.commit.author.name = "Author Two"
+        commit2.commit.author.email = "two@example.com"
+        mock_pr.get_commits.return_value = [commit1, commit2]
+
+        async def run_command_side_effect(command: str, **kwargs: Any) -> tuple[bool, str, str]:
```

Ruff (0.15.10)

[warning] Unused function argument: `kwargs`

(ARG001)